src/west/app/project.py

```diff
@@ -844,6 +844,53 @@ def do_run(self, args, user_args):
                 failed.append(project)
         self._handle_failed(args, failed)
 
+        # List projects not tracked by manifest.
+        projects = [project.abspath for project in self._cloned_projects(args, only_active=True)]
+        topdir = Path(self.topdir)
+        untracked = []
+
+        self.do_find_untracked(topdir, projects, untracked)
+
+        ignored_config = self.config.get('status.ignore')
+        ignored = ignored_config.split(',') if ignored_config else []
```

Comment:
We need a separate `.westignore` file for this for consistency with git and others. This will let users put each pattern on a separate line and let them ignore files with commas in their names.